### ISSUE TITLE
ECOS missing on new homepage

### ISSUE THREAD

**Issue by Isitar** [embotech/ecos#175]
Hello

The ecos link aswell as the "How to use" is missing on your new homepage.
https://www.embotech.com/ECOS/
https://www.embotech.com/ECOS/How-to-use/C-API

since you didn't provide a wiki here on github it is quite hard to find documentation about ecos.

**Comment by adomahidi** [comment]
Thanks for letting us know, we will post a solution to this asap here.

**Comment by adomahidi** [comment]
Temporarily available here until we move it to either the Github Wiki or to the new webpage

http://176.10.116.53/ECOS
http://176.10.116.53/ECOS/How-to-use
http://176.10.116.53/ECOS/How-to-use/C-API

**Comment by marcelino-pensa** [comment]
Hi @adomahidi, since I can't navigate through the tutorials (only the links above), could you please clarify on what is "Gpr, Gjc, Gir"? I understand what the G matrix is, but not really these partitions.
Is there any other source of documentation that might clarify this?

**Comment by adomahidi** [comment]
These arrays are the Column Compressed Storage format for G, which is a sparse matrix. See e.g. http://netlib.org/linalg/html_templates/node92.html

Von meinem iPhone gesendet

Am 01.07.2019 um 22:41 schrieb marcelino-pensa <notifications@github.com<mailto:notifications@github.com>>:


Hi @adomahidi<https://github.com/adomahidi>, since I can't navigate through the tutorials (only the links above), could you please clarify on what is "Gpr, Gjc, Gir"? I understand what the G matrix is, but not really these partitions.
Is there any other source of documentation that might clarify this?

**Comment by akapocsi** [comment]
Where should we go to find documentation on ECOS?

Is [http://176.10.116.53/ECOS/How-to-use](http://176.10.116.53/ECOS/How-to-use) still the recommended location?

It looks like https://www.embotech.com/ECOS/ is still missing.

**Comment by smerkli** [comment]
> Is http://176.10.116.53/ECOS/How-to-use still the recommended location?

For the moment, yes. I'm working on moving it now and will post here once the new location is ready. (Edit: Still ongoing)

**Comment by akapocsi** [comment]
Thanks Sandro!

**Comment by smerkli** [comment]
@akapocsi What I found in terms of documentation is now in the wiki of the repo here, https://github.com/embotech/ecos/wiki. Let me know if it's missing anything and I'll add it as well, otherwise I'll be closing this issue soon.

**Comment by pierre-haessig** [comment]
Thanks for making the docs available again. I'm not familiar with the old embotech page to assess the completeness of the wiki. Updating the links in the README should be the "finishing touch" of closing this issue.

**Comment by smerkli** [comment]
Good point, I've changed the links. Let me know if you spot other things you'd like to be added.
